network-api/networkapi/templates/fragments/buyersguide/article_listing_compact.html

```diff
@@ -0,0 +1,34 @@
+{% load i18n static wagtailcore_tags wagtailimages_tags %}
+
+<div>
+  {% if short_waves %}
+    <div class="tw-max-w-[11rem]">
+      {% include "./wavy_line.html" %}
+    </div>
+  {% else %}
+    {% include "./wavy_line.html" %}
+  {% endif %}
+
+  <h2 class="tw-font-zilla tw-font-normal tw-text-2xl tw-text-black tw-mt-2 tw-mb-[20px]">{{ heading }}</h2>
+
+  <ul class="tw-list-none {% if show_images %} tw-space-y-4 {% else %} tw-space-y-5 {% endif %} tw-p-0">
+    {% for article in articles  %}
+      <li class="tw-m-0" >
+        <a href="{% pageurl article %}" class="tw-text-black tw-flex hover:tw-text-blue-40 tw-items-center tw-no-underline">
+          {% if show_images %}
+            {% image article.hero_image fill-156x156 as img %}
+            <img src="{{ img.url }}" alt="{{ img.alt }}" class="tw-w-[78px] tw-h-[78px] tw-rounded-2xl tw-mr-4"/>
+          {% endif %}
+
+          <span class="tw-font-sans tw-font-bold tw-text-lg tw-leading-5">{{ article.title }}</span>
+        </a>
+      </li>
+    {% endfor %}
+  </ul>
```

Comment:
**Suggestion**

Should this be wrapped in `if articles`?

Reply:
Yea, good call. I thought I got them all 😅

Reply:
Updated in 9269f0c779cfb4028a0c6b081f2f612167a3661d